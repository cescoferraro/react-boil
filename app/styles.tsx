export const HomeStyle = require("./css/app.pcss")
