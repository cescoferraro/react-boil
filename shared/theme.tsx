const spacing = {
    iconSize: 24,
    desktopGutter: 24,
    desktopGutterMore: 32,
    desktopGutterLess: 16,
    desktopGutterMini: 8,
    desktopKeylineIncrement: 64,
    desktopDropDownMenuItemHeight: 32,
    desktopDropDownMenuFontSize: 15,
    desktopDrawerMenuItemHeight: 48,
    desktopSubheaderHeight: 48,
    desktopToolbarHeight: 56,
};
/**
 *  Light Theme is the default theme used in material-ui. It is guaranteed to
 *  have all theme variables needed for every component. Variables not defined
 *  in a custom theme will default to these values.
 */
export const BoilTheme = {
    spacing: spacing,
    fontFamily: 'Roboto, sans-serif',
    borderRadius: 2,
    palette: {
        primary1Color: "#303F9F",
        primary2Color: "#3F51B5",
        primary3Color: "#C5CAE9",
        accent1Color: "#C5CAE9",
        accent2Color: "#C5CAE9",
        accent3Color: "#C5CAE9",
        textColor: "#212121",
        secondaryTextColor: " #757575",
        /* alternateTextColor: darkBlack,*/
        /* canvasColor: white,*/
        borderColor: "#BDBDBD",
        disabledColor: "#BDBDBD",
        /* pickerHeaderColor: cyan500,*/
        /* clockCircleColor: fade(darkBlack, 0.07),*/
        /* shadowColor: fullBlack,*/
    },
};
