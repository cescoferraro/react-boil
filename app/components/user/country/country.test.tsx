import * as React from "react"
import Country from "./country"
import * as renderer from 'react-test-renderer';
import { MuiThemeProvider } from 'material-ui/styles';
import { theme } from "../../../../shared/theme/index";
test("Render an Item", () => {
    const component = renderer.create(
        <MuiThemeProvider theme={theme}>
            <Country
                profile={{ nat: "BR" }}
                className="coolClassName"
            />
        </MuiThemeProvider>
    )
    expect(component).toMatchSnapshot()
})
