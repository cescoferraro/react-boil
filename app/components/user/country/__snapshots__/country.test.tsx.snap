// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render an Item 1`] = `
<div
  className="item"
>
  <div
    className="itemIcon"
  >
    <img
      alt="Canada Flag"
      className="coolClassName"
      height={undefined}
      src="/flags/flags-iso/flat/32/BR.png"
      width={undefined}
    />
  </div>
  <div
    className="itemText"
  >
    <a
      className="key"
    >
       
      Nationality
       
    </a>
    <a
      className="value"
    >
       
      BR
       
    </a>
  </div>
</div>
`;
